docs/modules/workflow/con-ploigos-workflow-infrastructure.adoc

```diff
@@ -1,7 +1,61 @@
 [id="{ProjectNameID}-workflow-infrastructure", reftext="{ProjectName} Infrastructure Requirements"]
 = Infrastructure Requirements 
 
-Reference information on the infrastructure used to develop and test the MVP of the <<{ProjectNameID}-workflow-assembly-{ProjectNameID}-workflow>>.
+Below is the infrastructure used to develop and test the Minimum Viable Product(MVP) of the Ploigos CI/CD Workflow Reference Implementation. The most basic requirement for the deployment of the Ploigos CI/CD Workflow is an operational OCP 4.x cluster. 
+
+.Red Hat OpenShift Sizing
+[cols="a,a,a,a,a,a",options="header"]
+|===
+| Node
+| CPUs
+| Memory (GB)
+| Disk (GB)
+| AWS EC2 Instance Type
+| Sizing Source
+
+| Master 0
```

Comment:
@twling change `master` to `control`

Reply:
Easy enough to change, but would this be confusing until openshift changes their terminology?